### PR TITLE
cn translation by hengrui

### PR DESCRIPTION
**Pull request by HENGRui6**
翻译了oneflow.sub, oneflow.exp, oneflow.log1p, oneflow.ceil, oneflow.atan, oneflow.asinh, oneflow.asin, oneflow.reciprocal, oneflow.mul, oneflow.div, oneflow.abs
[oneflow.sub](https://github.com/Oneflow-Inc/oneflow/blob/cd5294e4bf5a2758d92f7ca1c2f5a6e0276b6ddb/python/oneflow/framework/docstr/math_ops.py#L183)
b2b048852de356b153fe16f4c294b6e

[oneflow.abs](https://github.com/Oneflow-Inc/oneflow/blob/cd5294e4bf5a2758d92f7ca1c2f5a6e0276b6ddb/python/oneflow/framework/docstr/math_ops.py#L20)
52df6dbd667716cd018f1eb8e887298

[oneflow.div](https://github.com/Oneflow-Inc/oneflow/blob/cd5294e4bf5a2758d92f7ca1c2f5a6e0276b6ddb/python/oneflow/framework/docstr/math_ops.py#L80)
7e6bce0d82291ac0e008c4e6c3d7db3

[oneflow.mul](https://github.com/Oneflow-Inc/oneflow/blob/cd5294e4bf5a2758d92f7ca1c2f5a6e0276b6ddb/python/oneflow/framework/docstr/math_ops.py#L123)
0bb2fabdbe01903e90a5a9ea4145de7

[oneflow.reciprocal](https://github.com/Oneflow-Inc/oneflow/blob/cd5294e4bf5a2758d92f7ca1c2f5a6e0276b6ddb/python/oneflow/framework/docstr/math_ops.py#L163)
3abbbe05cd5291c9ba0dd84e8c0db0f

[oneflow.asin](https://github.com/Oneflow-Inc/oneflow/blob/cd5294e4bf5a2758d92f7ca1c2f5a6e0276b6ddb/python/oneflow/framework/docstr/math_ops.py#L222)
3ca7d182cb2adc6da0f2410571d766c

[oneflow.asinh](https://github.com/Oneflow-Inc/oneflow/blob/cd5294e4bf5a2758d92f7ca1c2f5a6e0276b6ddb/python/oneflow/framework/docstr/math_ops.py#L255)
56f17dd8e4c4e1aeda41e56ad4422ef

[oneflow.atan](https://github.com/Oneflow-Inc/oneflow/blob/cd5294e4bf5a2758d92f7ca1c2f5a6e0276b6ddb/python/oneflow/framework/docstr/math_ops.py#L290)
5bb561ddd6622248c6b1336f95d7223

[oneflow.ceil](https://github.com/Oneflow-Inc/oneflow/blob/cd5294e4bf5a2758d92f7ca1c2f5a6e0276b6ddb/python/oneflow/framework/docstr/math_ops.py#L315)
1c5951fe14cfd379b5f025f9ee7ca33

[oneflow.log1p](https://github.com/Oneflow-Inc/oneflow/blob/cd5294e4bf5a2758d92f7ca1c2f5a6e0276b6ddb/python/oneflow/framework/docstr/math_ops.py#L365)
d7589ce4e0160b831d63cab1328bc1f

[oneflow.exp](https://github.com/Oneflow-Inc/oneflow/blob/cd5294e4bf5a2758d92f7ca1c2f5a6e0276b6ddb/python/oneflow/framework/docstr/math_ops.py#L386)
42367c8f78bf5f9656a3c9f12cae514

增加了对acos, acosh, atanh, sign, sinh, tan, clamp, cos, cosh 的翻译， 
[oneflow.acos](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L419)
4de1b303fd72b688363394b2528be96

[oneflow.acosh](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L445)
553dd33033abfa8ced11957edaa5901

[oneflow.atanh](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L474)
42caae98fef78f6d0cca3fafba33908

[oneflow.sign](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L499)
f22c77bee0659a645147f1a41a0e5b9

[oneflow.sinh](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L558)
b5ee7a537b8e82951314819362a1dd6

[oneflow.tan](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L589)
561e013d02e3090014cf605cd31b22e

[oneflow.clamp](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L614)
abc9b6cceeb11f5e14c7b6a047146f3

[oneflow.cos](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L664)
d52a9184b569435f700e0bae102bd15

[oneflow.cosh](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L688)
bd512ecf97f4454315f199c6a97913d

[oneflow.erf](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L715)
aff2b258730749f52e84bedf01989e9

[oneflow.erfc](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L762)
940d8207dbdbc397cda6c41df14fbc8

[oneflow.expm1](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L797)
6d5f3dffc3434609d7036503738f9c4

[oneflow.fmod](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L842)
9c74a12eeefc55d25655ed7206a33fb

[oneflow.log](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L875)
6270b10c3d793cde9b86447d0ef1359

[oneflow.minimum](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L900)

[oneflow.maximum](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L923)

[oneflow.pow](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L946)
459b4402792e7e95a11db1e00663fcd

[oneflow.rsqrt](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L987)

[oneflow.sqrt](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L1012)

[oneflow.square](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L1038)
c213a218d5718af2ecb713aaba83a94

[oneflow.matmul](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L1063)
9f29011ba9f0a44af5855dbf83ae4d7

[oneflow.round](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L1089)
cb370ad95cf85e8d70c5483a7180e5b

### CODE REVIEW

File: docs/source/cn/math_ops.py [hunk 1]
```diff
@@ -41,3 +41,40 @@
 
     """,
 )
+
+reset_docstr(oneflow.sub, """计算'input'和'other'的差，支持 element-wise、标量和广播形式的加法。
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:45]: 参考上面 `add` 的例子：第一行写算子的 signature，这样 html 会显示你写的 signature。
input 和 other 等不要贸然把反引号改成引号（反引号在 rst 中有起特殊意义），需要改回去。

中文文档的显示效果应该在 PR description 那里截图显示（reviewer 看docstring 源码也很难猜html的样式）。

**用 r 字符串更好**（这个视频中提醒过一次……）

代码最好用 `python -m black 你修改的文件.py` 格式化后再提交。

Comment (HENGRui6) [reply docs/source/cn/math_ops.py:45]: 明白

**Comment by doombeaker** [comment]
从抬杠的角度说，“翻译了oneflow.sub的中文版” 本身是病句……
应该是 “翻译了oneflow.sub”，或者 “oneflow.sub的中文版”。

做API文档工作需要很细致，否则作者和 reviewer 都会很痛苦。

File: docs/source/cn/math_ops.py [hunk 2]
```diff
@@ -41,3 +41,376 @@
 
     """,
 )
+
+reset_docstr(
+    oneflow.sub,
+    r"""sub(input, other)
+    
+    计算 `input` 和 `other` 的差，支持 element-wise、标量和广播形式的加法。
```
Comment (hjchen2) [inline docs/source/cn/math_ops.py:49]: 加法 -> 减法

Comment (HENGRui6) [reply docs/source/cn/math_ops.py:49]: ok我全部重新检查一下

**Comment by doombeaker** [comment]
截图里的有些明显是有格式问题的……

File: docs/source/cn/math_ops.py [hunk 3]
```diff
@@ -41,3 +41,376 @@
...
+)
+
+reset_docstr(
+    oneflow.asinh,
+    r"""asinh()
+    
+    返回一个包含 :attr:`input` 中每个元素的反双曲正弦的新 tensor。
+
+    .. math::
+        \\text{out}_{i} = \\sinh^{-1}(\\text{input}_{i})
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:257]: 如果使用r字符串，字符是没有转义的，所以不需要两个 `\` ，如果真输入2个（`\\`），在 latex 语法里意味着换行。
```suggestion
        \text{out}_{i} = \sinh^{-1}(\text{input}_{i})
```

不过输入错误是没什么关系的。你这里的问题是，自己的截图中有明显的排版问题，自己难道无法发现的吗？（既没有修正，也没有求助，就放在那里等别人review）

File: docs/source/cn/math_ops.py [hunk 4]
```diff
@@ -41,3 +41,376 @@
...
+)
+
+reset_docstr(
+    oneflow.atan,
+    r"""atan(input)
+
+    返回一个包含 :attr:`input` 中所有元素的反正切的新 tensor。
+
+    .. math::
+        \\text{out}_{i} = \\tan^{-1}(\\text{input}_{i})
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:293]: 同上，其它的请自己检查

**Comment by doombeaker** [comment]
还有个建议，为了方便review翻译内容的质量，在 PR Description 里附上下原有 docstr 的代码链接吧。
示例：
[oneflow.asinh](https://github.com/Oneflow-Inc/oneflow/blob/master/python/oneflow/nn/modules/math_ops.py#L287)

File: docs/source/cn/math_ops.py (hunk 4 above)
Comment (HENGRui6) [reply docs/source/cn/math_ops.py]: 已全部修改

Comment (HENGRui6) [reply docs/source/cn/math_ops.py]: 已修改

File: docs/source/cn/math_ops.py [hunk 5]
```diff
@@ -41,3 +41,695 @@
...
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> import numpy as np
+        >>> x1 = flow.tensor(np.array([2, 3, 4]).astype(np.float32))
+        >>> out1 = flow.acosh(x1)
+        >>> out1
+        tensor([1.3170, 1.7627, 2.0634], dtype=oneflow.float32)
+        >>> x2 = flow.tensor(np.array([1.5, 2.6, 3.7]).astype(np.float32),device=flow.device('cuda'))
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:470]: 因为现在的测试是用的 github 的免费计算资源，没有 GPU，所以这里过不了 CI。
@jackalcooper 可否帮忙支持下 GPU 的CI测试，我的理解是设置好权限，让CI在我们自己的服务器上就可以了？

**Comment by doombeaker** [comment]
截图的格式有问题。可能是因为截图时，没有完整下载整个 html 目录？而只下载了单独的页面？导致某些 css 样式没下载下来？因为我发现代码也是没有高亮的，这不正常。

File: docs/source/cn/math_ops.py [hunk 6]
```diff
@@ -41,3 +41,695 @@
...
+    
+    计算 `input` 除以 `other`，支持 element-wise、标量和广播形式的除法。
+    公式为：
+
+    .. math::
+        out = \frac{input}{other}
+    
+    参数：
+        input (Union[int, float, oneflow.Tensor]): input.
+        other (Union[int, float, oneflow.Tensor]): other.
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:121]:
```suggestion
        - input (Union[int, float, oneflow.Tensor]): input.
        - other (Union[int, float, oneflow.Tensor]): other.
```
不加 `-` 的话，这两个就合并成同一行了（请自己严格检查编译结果）

**Comment by HENGRui6** [comment]
增加了对acos, acosh, atanh, sign, sinh, tan, clamp, cos, cosh 的翻译， 
[oneflow.acos](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L419)
01030cb05baa8487d112ebe8bc58e2a

[oneflow.acosh](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L445)
00dabc1a701874118e3f7ce9811cb9e

[oneflow.atanh](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L474)
42caae98fef78f6d0cca3fafba33908

[oneflow.sign](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L499)
image

[oneflow.sinh](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L558)
image

[oneflow.tan](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L589)
f08a0d1a1ac92675d1816ab2000ab05

[oneflow.clamp](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L614)
7726dc64769313a6f8d650da9e68d78

[oneflow.cos](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L664)
ab5f319a367cd0237b9df8a80320970

[oneflow.cosh](https://github.com/Oneflow-Inc/oneflow/blob/beeed4ee5c2b0a4b19aafe2f74e8598ab554c97f/python/oneflow/framework/docstr/math_ops.py#L688)
c73ffc29b0d1293e7076275ccc907e4

File: docs/source/cn/math_ops.py [hunk 7]
```diff
@@ -41,3 +41,695 @@
...
+        >>> out = flow.mul(input,other).numpy()
+        >>> out.shape 
+        (2, 3)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.reciprocal,
+    r"""reciprocal(x: Tensor) -> Tensor
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:198]: 翻译的过程中可以顺便帮忙多做一个对齐工作的事情。背景是：oneflow 的有些算子的参数名，其实是没有和 torch 对齐的。比如这个 reciprocal，参数名是 `x`，但是实际上 torch 的是 `input` （torch 的第一个参数几乎都是 `input`） https://pytorch.org/docs/stable/generated/torch.reciprocal.html?highlight=reciprocal#torch.reciprocal 。

你在翻译的过程中，发现可疑的，就去搜一下 torch 的 API 文档。如果确实没有对齐，就记录以下。最后更新到这个 issue里：https://github.com/Oneflow-Inc/oneflow/issues/6003 这样之后会有人更新对齐的（当然如果能自己直接去给 oneflow 仓库提交 PR 修改，就更好了）。

File: docs/source/cn/math_ops.py [hunk 8]
```diff
@@ -41,3 +41,695 @@
...
+                 [14., 15., 16.]]], dtype=oneflow.float32)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.log1p,
+    r"""log1p(input: Tensor) -> Tensor
+    
+    返回一个新的 tensor，其自然对数为 (1 + input)。
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:366]: 其自然对数为 (1 + input) 的表述应该是不准确的。直接说“公式为”吧

File: docs/source/cn/math_ops.py [hunk 9]
```diff
@@ -41,3 +41,695 @@
...
+        >>> out = flow.log1p(x)
+        >>> out
+        tensor([0.8329, 0.9163, 1.3083], dtype=oneflow.float32)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.exp,
+    r"""exp(x: Tensor) -> Tensor
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:387]: 同上，翻译过程中会发现，其实我们 exp 的参数名字没和 torch 对齐。

File: docs/source/cn/math_ops.py (hunk 6 above)
Comment (HENGRui6) [reply docs/source/cn/math_ops.py:121]: 已解决

File: docs/source/cn/math_ops.py (hunk 8 above)
Comment (HENGRui6) [reply docs/source/cn/math_ops.py:366]: 已解决

File: docs/source/cn/math_ops.py (hunk 9 above)
Comment (HENGRui6) [reply docs/source/cn/math_ops.py:387]: 已修改

File: docs/source/cn/math_ops.py (hunk 7 above)
Comment (HENGRui6) [reply docs/source/cn/math_ops.py:198]: 已解决中文版，会去尝试解决英文版

File: docs/source/cn/math_ops.py [hunk 10]
```diff
@@ -3,14 +3,25 @@
 
 reset_docstr(
     oneflow.add,
-    r"""add(input, other)
+    r"""add(input, other, *, out=None) -> Tensor
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:6]: …… 我们并不支持这个原型。我们现在只支持 add(input, other)

**Comment by HENGRui6** [comment]
> 截图里的公式格式有问题（全部向右对齐了）。可能是因为截图时，没有完整下载整个 html 目录？而只下载了单独的页面？导致某些 css 样式没下载下来？因为我发现代码也是没有高亮的，这不正常。

确实是没下载全，新的截图已解决此问题

File: docs/source/cn/math_ops.py (hunk 8 above)
Comment (HENGRui6) [reply docs/source/cn/math_ops.py]: 已解决

File: docs/source/cn/math_ops.py [hunk 11]
```diff
@@ -41,3 +49,1090 @@
...
+
+    返回值：
+        oneflow.Tensor: 结果张量
+    
+    示例：
+
+    .. code-block:: python
+
+        >>> import oneflow as f  
+        >>> arr = np.random.randn(2, 3, 4, 5)
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:1022]: 这个 docstr 是错的，因为其实没有导入 np 库，这里会报错。 英文版原版也是这样写的吗？那需要备注下，之后也要把 oneflow 仓库里的 英文版本修正下。

Comment (HENGRui6) [reply docs/source/cn/math_ops.py:1019]: 已修改

File: docs/source/cn/math_ops.py [hunk 12]
```diff
@@ -41,3 +49,1208 @@
...
+    
+    返回一个Tensor, 其元素为用 `exponent` 计算 `input` 中
+    每​​个元素的幂。`exponent` 可以是单个浮点数，整数或者与 
+    `input` 具有相同形状的 tensor。
+
+    当指数是标量时，操作为：
+
+    .. math::
+        \text{out}_i = x_i ^ \text{exponent}
+\u200b
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:1103]: 这个是什么

Comment (HENGRui6) [reply docs/source/cn/math_ops.py]: 这个原英文版里面有，我怕出问题就留下了，我也不知道是什么

File: docs/source/cn/math_ops.py [hunk 13]
```diff
@@ -3,41 +3,1196 @@
...
+
+    .. math::
+        out = \frac{input}{other}
+    
+    参数：
+        - **input** (Union[int, float, oneflow.Tensor]): input.
+        - **other** (Union[int, float, oneflow.Tensor]): other.
+
+    返回值：
+        oneflow.Tensor: 结果张量
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:141]:
```suggestion
        oneflow.Tensor
```

Comment (doombeaker) [reply docs/source/cn/math_ops.py:141]: 我看了下，其他的中文版，以及英文原版，没有专门给返回值做说明。
我们约定下，如果比较简单的单返回值，就给个类型就可以了吧。如果遇到复杂的多个返回值的情况，我们就再讨论。

这里个pr里上下有关的，都统一改一下

Comment (doombeaker) [inline docs/source/cn/math_ops.py:141]:
```suggestion
    返回类型：
        oneflow.Tensor
```

我看了下，其他的pytorch中文版，以及oneflow英文原版，大部分没有专门给返回值做说明。
我们约定下，如果比较简单的单返回值，就给个返回类型就可以了吧。如果遇到复杂的多个返回值的情况，我们就再讨论。

这里个pr里上下有关的，都统一改一下。

注意，返回值和返回类型时不一样的。

File: docs/source/cn/math_ops.py [hunk 14]
```diff
@@ -3,41 +3,1196 @@
...
+    公式为：
+
+    .. math::
+        \text{out}_{i} = \sinh^{-1}(\text{input}_{i})
+
+    参数：
+        **x** (Tensor): 输入张量
+
+    关键词参数：
+        **out** (Tensor, optional): 输出张量
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:300]: 没有 out，文档这里的说明是多余的。其他地方也请检查。

File: docs/source/cn/math_ops.py [hunk 15]
```diff
@@ -3,41 +3,1196 @@
...
+                 [14., 15., 16.]]], dtype=oneflow.float32)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.log1p,
+    r"""log1p(x) -> Tensor
+    
+    返回一个新的 tensor，其自然对数的公式为 (1 + x)。
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:415]:
```suggestion
    返回一个新的 tensor，计算自然对数，公式为：
```

…… 这个已经改过一次了，有双方沟通理解偏差的问题……互相思考下下次怎么避免。

File: docs/source/cn/math_ops.py [hunk 16]
```diff
@@ -3,41 +3,1196 @@
...
+    示例：
+
+    .. code-block:: python 
+
+        >>> import oneflow as flow
+
+        >>> flow.fmod(flow.tensor([-3., -2, -1, 1, 2, 3], dtype=flow.float32), 2.)
+        tensor([-1., -0., -1.,  1.,  0.,  1.], dtype=oneflow.float32)
+        >>> flow.fmod(flow.tensor([1, 2, 3, 4, 5.], dtype=flow.float32), 1.5)
+        tensor([1.0000, 0.5000, 0.0000, 1.0000, 0.5000], dtype=oneflow.float32)
```
Comment (doombeaker) [inline docs/source/cn/math_ops.py:940]:
```suggestion
        tensor([1.0, 0.5, 0.0, 1.0, 0.5], dtype=oneflow.float32)
```

**Review by doombeaker — APPROVED** [review]

File: docs/source/cn/math_ops.py (hunk 16 above)
Comment (doombeaker) [reply docs/source/cn/math_ops.py:940]: 算了，这个不用改了，和英文版本先一致吧

**Pull request merged**